src/progress-steps/styled-components.js

```diff
@@ -55,6 +57,12 @@ export const StyledIcon = styled<StylePropsT>(
 
     const marginTop = `calc((${font.lineHeight} - ${size}) / 2)`;
 
+    if ($theme.direction && $theme.direction === 'rtl') {
```

Comment:
should we just set the default in the theme to `ltr`, so this conditional is simpler?

Reply:
`auto` would make more sense if using a default value.